docs/dev/Kibana-Reporting-Design-Proposal.md

```diff
@@ -37,6 +37,7 @@ Kibana Reports for Open Distro allows ‘Report Owner’ (engineers, including b
 1. As a user, I can download a PDF, PNG, CSV, XLS file in-context from any available panel (e.g. Dashboards, visualizations, saved searches, etc).
 1. As a user, any time I download a report from any available panel (e.g. Dashboards, visualizations, saved searches, etc), a report will be auto-generated in the reports list.
 1. As a user, I can create a report definition from the share menu of any available panel (e.g. Dashboards, visualizations, saved searches, etc).
+1. As a user, I can download a report in-context only if my search, visualizations, dashboards is saved.
```

Comment:
This is not true, even when a dashboard is not saved, we should be able to generate a report file by clicking the download button

Reply:
ok